html/includes/modal/alert_template.inc.php

```diff
@@ -225,11 +225,8 @@ function alertTemplateAjaxOps(template, name, template_id, title, title_rec)
         dataType: "html",
         success: function(msg){
             if(msg.indexOf("ERROR:") <= -1) {
-                $("#message").html('<div class="alert alert-info">'+msg+'</div>');
+                toastr.success(msg);
                 $("#alert-template").modal('hide');
-                setTimeout(function() {
```

Comment:
It's not a huge thing but this stops the page reloading which means changing the template name won't show in the webui.

You could just update the row with the new name to avoid the reload, it's the only element that would change.

Otherwise, we'll be doing an overhaul on the templating system so you could leave this PR until that's done and we can take care of it all then.

Reply:
If this is going to create extra work for you I'm happy to close the PR. I'll wait for the overhaul, and I can fix it afterwards as well, doesn't matter.

Reply:
It's no extra work but for this PR to be merged you either need to re-add the setTimeout call or dynamically update the templates table with the updated template name after editing (the former is easier and fine)